docs/quickstart/quickstart_chains/humanode.md

```diff
@@ -61,7 +61,7 @@ Now that you have made essential changes to the GraphQL Schema file let’s move
 
 ## 2. Updating Your Project Manifest File
 
-The Project Manifest (`project.yaml`) file works as an entry point to your project. It defines most of the details on how SubQuery will index and transform the chain data. For Substrate/Polkadot chains, there are three types of mapping handlers (and you can have more than one in each project):
+The Project Manifest (`project.yaml`) file works as an entry point to your project. It defines most of the details on how SubQuery will index and transform the chain data. For Humanode chain (and Substrate/Polkadot chains in general), there are three types of mapping handlers (and you can have more than one in each project):
```

Comment:
For  Humandoe chains

Reply:
No, `Humanode` is correct

Reply:
Did you mean to put `chains` in plural? That's a good idea

Reply:
Apologies. Yes,I meant plural :)